test/e2e/gorillamux/traces.json

```diff
@@ -0,0 +1,52 @@
+{
+  "resourceSpans": [
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "service.name",
+            "value": {
+              "stringValue": "sample-app"
+            }
+          },
+          {
+            "key": "telemetry.sdk.language",
+            "value": {
+              "stringValue": "go"
+            }
+          }
+        ]
+      },
+      "scopeSpans": [
+        {
+          "scope": {
+            "name": "github.com/gorilla/mux"
+          },
+          "spans": [
+            {
+              "attributes": [
+                {
+                  "key": "http.method",
+                  "value": {
+                    "stringValue": ""
+                  }
+                },
+                {
+                  "key": "http.target",
+                  "value": {
+                    "stringValue": ""
+                  }
+                }
+              ],
+              "kind": 2,
+              "parentSpanId": "",
+              "spanId": "",
+              "status": {},
+              "traceId": ""
```

Comment:
I was surprised that spanId and traceId were empty here. ping @edeNFed does that look right?

Reply:
Ah I didn't enable a propagator. Setting `OTEL_PROPAGATORS` on the sample app fills these in (and the request fields too).

Reply:
hm but this empty span is still here

Reply:
I think this empty span is a bug in the auto-instrumentation, but that should be a separate issue to track instead of fix in this PR